bin/tag-release-pr.js

```diff
@@ -23,11 +24,17 @@ const callback = options => {
 	if (options.conflict) {
 		flow = filterFlowBasedOnDevelopBranch(options, prRebaseConflict);
 
+		if (options.scripts[`post${options.command}`]) {
```

Comment:
Wouldn't you need to do this logic for each command to support pre and post hooks?

I only see updates in the tag-release-pr.js and tag-release-qa.js files

Reply:
Also, is there anywhere higher this logic can be located so it doesn't need to be repeated?

Reply:
Yeah the global one is in setup.js. It does it there too but those are the files that have their own callbacks that changes the workflow so I have to add it there.

Reply:
Sounds good. This time I should prob test each of the pre and post hooks. I guess I wasn't very thorough last time 🤦‍♂